docs/testrunner-toolkit/bundling.md

```diff
@@ -0,0 +1,103 @@
+---
+id: bundling
+title: Bundling Test Files
+sidebar_label: CLI Reference
+---
+
+The [saucectl command line interface](saucectl) runs tests (Cypress, TestCafe, Playwright, ...) by bundling up your root directory, sending the bundle to the Sauce Labs cloud (or a Docker container), unpacking the bundle and then running the tests. This page explains how bundling works and how you can reduce the size of your bundle for improved performance.
```

Comment:
Might be good to call out that the Docker container is for your local or your own infra.